sdk/formrecognizer/Azure.AI.FormRecognizer/MigrationGuide.md

```diff
@@ -617,7 +617,7 @@ Train a custom model with `4.0.x`:
 Uri trainingFileUri = <trainingFileUri>;
 var client = new DocumentModelAdministrationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
-BuildModelOperation operation = await client.StartBuildModelAsync(trainingFileUri);
+BuildModelOperation operation = await client.StartBuildModelAsync(trainingFileUri, DocumentBuildMode.Template);
```

Comment:
I'm defaulting everything we had to `Template` for the time being since `Neural` takes too long to run (at least 10 mins). We can consider adding it to a sample or two if the service team or we believe it's important to showcase this scenario.

Reply:
By the way, I think it's important to address the build mode feature in the Migration Guide. It's a major breaking change and we want users to start thinking about which build mode has a better match with their documents.

I wanted to focus on the code complete work for Friday first, so I'll postpone the Migration Guide work to the next week. Here's the issue for tracking: https://github.com/Azure/azure-sdk-for-net/issues/26782

Reply:
@jsquire `Template` is the name of the model engine that has always existed. `Neural` is a new, more expensive (time and dollar cost) alternative that should yield higher-quality results in certain circumstances, so it's fair to assume that anywhere we previously didn't specify the build mode (because no such option existed), passing `Template` will yield equivalent results in the new API.